raiden/raiden_service.py

```diff
@@ -540,9 +541,67 @@ def connection_manager_for_token(self, token_address):
             raise InvalidAddress('token is not registered.')
         return manager
 
+    def leave_all_token_networks_async(self):
```

Comment:
Could you please us a different suffix? All but this method return a single `AsyncResult`, while this returns a list of async results, or change the return to an `AsyncResult` that is set after all the inner one are.